src/core/transform.js

```diff
@@ -222,8 +222,8 @@ p5.prototype.rotate = function(angle, axis) {
  * 3d box rotating around the x axis.
  */
 p5.prototype.rotateX = function(angle) {
-  p5._validateParameters('rotateX', arguments);
   this._assert3d('rotateX');
+  p5._validateParameters('rotateX', arguments);
```

Comment:
for example, I think these additions of validateParameters calls could be in a separate PR